src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -75,27 +75,27 @@ static AnimationHeader* D_80A7B918[] = {
     &object_in_Anim_01646C, &object_in_Anim_0175D0, &object_in_Anim_017B58, &object_in_Anim_018C38,
 };
 
-static Gfx* sAlternateOutfitDLs[] = {
+static Gfx* sAdultEraDLs[] = {
```

Comment:
I don't think I like "era"
just "adult" is enough I think, we know it would refer to link's age?